src/controllers/usersController.js

```diff
@@ -0,0 +1,77 @@
+const usersServices = require('../services/usersService.js');
+
+const get = async (req, res) => {
+  const users = await usersServices.getAllUsers();
+
+  res.send(users);
+};
+
+const getOne = async (req, res) => {
+  const { id } = req.params;
+  const user = await usersServices.getUserById(id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(user);
+};
+
+const create = async (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
```

Comment:
Consider adding a check to ensure that `name` is a valid string before attempting to create a user. This will prevent potential issues with invalid input.